### ISSUE TITLE
--noop/-n flag not honored

### ISSUE THREAD

**Issue by ss7548** [egnyte/gitlabform#108]
Running the following unexpectedly changes the group visibility settings:

**gitlabform --noop group-name**  or **gitlabform -n group-name**

Before the run:
![image](https://user-images.githubusercontent.com/15113378/85921861-5964eb00-b84d-11ea-88e9-a8729e2a9967.png)

After the run:
![image](https://user-images.githubusercontent.com/15113378/85921887-7d283100-b84d-11ea-863f-bd01008de2e0.png)

Here is the config:
```YAML
gitlab:
  url: https://gitlab.mydomain.com
  token: xxxyyyzzz
  api_version: 4

skip_groups:
  - lost-and-found

common_settings:
  group_settings:
    visibility: internal
  project_settings:
    visibility: internal
```

Here is the run:
```bash
$ gitlabform -v -n awscicd
Reading config from file: C:\users\ss7548\documents\git-repos\config.yml
Connected to GitLab version: 13.0.1-ee (7154da26005)
Config file is declared to be compatible with GitLab API v4
Reading config from file: C:\users\ss7548\documents\git-repos\config.yml
*** # of groups to process: 1
*** # of projects to process: 0
Setting group_settings
Setting group settings: {'visibility': 'internal'}
```

gitlabform 1.15.1
gitlab: 13.0.1
python: 3.8.3

**Comment by gdubicki** [comment]
Thanks for reporting this!

In fact `--noop` never worked for group configs, only for projects...

Will be fixed in 1.15.2, to be released tomorrow/the day after (fix is already in the master).

**Comment by gdubicki** [comment]
Sorry for the slight delay @ss7548 ! Please me know if you still have any issues.
